dipy/viz/ui.py

```diff
@@ -693,3 +700,523 @@ def re_align(self, window_size_change):
                              self.center[1] + window_size_change[1]))
         else:
             raise ValueError("You can only left-align or right-align objects in a panel.")
+
+
+class TextActor2D(vtkTextActor):
+    """ Inherits from the default vtkTextActor and helps setting the text.
+
+    Contains member functions for text formatting.
+
+    """
+    def message(self, text):
+        """ Set message after initialization.
+
+        Parameters
+        ----------
+        text : string
+            The message to be set.
+
+        """
+        self.SetInput(text)
+
+    def set_message(self, text):
+        """ Modify text message.
+
+        Parameters
+        ----------
+        text : string
+            The new message.
+
+        """
+        self.SetInput(text)
+
+    def get_message(self):
+        """ Gets message from the text.
+
+        Returns
+        -------
+        message : string
+            The current text message.
+
+        """
+        return self.GetInput()
+
```

Comment:
Agreed, it's a more Pythonic way. I gave it a try but the unit testing framework errored, probably because of some VTK support. I can try doing so in a later version.